pages/dashboard/github-integration.md

```diff
@@ -0,0 +1,153 @@
+# Integração com o GitHub
+
+O [Cypress Dashboard](https://on.cypress.io/dashboard) consegue integrar seus testes Cypress
+com seu Workflow do GitHub através de [verificações de status](#verificacoes-de-status)
+e comentários de [pull requests](#comentarios-de-pull-requests). 
+
+Um projeto inicialmente precisa estar [configurado para gravação](https://docs.cypress.io/guides/dashboard/projects)
+para que o Cypress Dashboard possa utilizar a integração com o GitHub.
+
+![pull-request-cypress-integration-comments-github-checks](https://docs.cypress.io/_nuxt/img/pull-request-cypress-integration-comments-github-checks.8fd68f7.jpg)
+
+>A plataforma local GitHub Enterprise atualmente não é suportada.
+
+>A Integração com o GitHub depende que o seu ambiente de integração contínua forneceça dados SHA de commit
+(típicamente via variável de ambiente). Isso não é um problema para a maioria dos usuários,
+mas se você estiver com problemas de integração com o GitHub com sua configuração de integração contínua,
+por favor, certifique-se que a informação git está sendo enviada corretamente
+seguindo [essas diretrizes](https://docs.cypress.io/guides/continuous-integration/introduction#Git-information).
+Se após isso você ainda estiver enfrentando problemas, por favor, entre em [contato conosco](mailto:hello@cypress.io).
+
+## Instalando o app Cypress GitHub
+
+Antes que você possa habilitar a integração com o GitHub em seus projetos Cypress, você primeiro deve instalar
+o app Cypress GitHub. Você pode iniciar o processo de instalação do aplicativo através da página de configuração
+da sua organização, ou a partir de uma página de configuração de um projeto no [Cypress Dashboard](https://on.cypress.io/dashboard).
+
+### Instalando via configurações de integração da organização
+
+1. Vá para a [Página de organizações](https://dashboard.cypress.io/organizations)no dashboard, ou abra a lista de organizações.
+2. Selecione a organização que você deseja integrar com uma conta do GitHub ou organização do GitHub.
+![Docs Cypress](https://docs.cypress.io/_nuxt/img/select-cypress-organization.41ec503.png)
+3. Visite a página de **integrações** da organização selecionada, na navegação lateral.
+![Docs Cypress](https://docs.cypress.io/_nuxt/img/navigate-to-organization-integrations.0c43d75.png)
+4. CLique no botão **Instalar Integração com o GitHub**.
+
+### Instalando via configurações do projeto
+
+1. Selecione sua organização na lista de organizações.
+![Docs Cypress](https://docs.cypress.io/_nuxt/img/select-cypress-organization.41ec503.png)
+2. Selecione o projeto que você deseja integrar a um repositório GitHub.
+[Docs Cypress](https://docs.cypress.io/_nuxt/img/select-cypress-project.fe3b44b.png)
+3. Vá para a página de configurações do projeto.
+[Docs Cypress](https://docs.cypress.io/_nuxt/img/visit-project-settings.43a21a4.png)
+4. Role até a seção **Integração com o GitHub**.
+5. Clique no botão **Instalar Cypress GitHub App**.
```

Comment:
O ideal seria você traduzir apenas o texto e deixar o nome de abas, botões ou elementos do Cypress Dashboard em inglês mesmo para o pessoal não se confudir quando for reproduzir.